src/app/components/user/UserUtil.js

```diff
@@ -23,7 +22,7 @@ const messages = defineMessages({
 });
 
 const LocalizedRole = ({ role, children }) => role ?
-  <FormattedMessage {...messages[role]}>{children}</FormattedMessage>
+  <FormattedMessage {...messages[role]}>{children}</FormattedMessage> // eslint-disable-line @calm/react-intl/missing-attribute
```

Comment:
Note to reviewers: I am not sure how to resolve this, or if its necessary to. Any guidance would be appreciated